packages/authorization-process/test/createApiClient.test.ts

```diff
@@ -0,0 +1,77 @@
+import { describe, it, vi, beforeAll, afterAll, expect } from "vitest";
+import {
+  Client,
+  ClientAddedV2,
+  UserId,
+  clientKind,
+  generateId,
+  toClientV2,
+  unsafeBrandId,
+} from "pagopa-interop-models";
+import { genericLogger } from "pagopa-interop-commons";
+import {
+  decodeProtobufPayload,
+  readLastEventByStreamId,
+} from "pagopa-interop-commons-test/index.js";
+import { ApiClientSeed } from "../src/model/domain/models.js";
+import { authorizationService, postgresDB } from "./utils.js";
+
+describe("createConsumerClient", () => {
+  const userId = generateId();
+
+  beforeAll(async () => {
+    vi.useFakeTimers();
+    vi.setSystemTime(new Date());
+  });
+
+  afterAll(() => {
+    vi.useRealTimers();
+  });
+
+  const clientSeed: ApiClientSeed = {
+    name: "Seed name",
+    description: "Description",
+    members: [userId],
+  };
+  it("should write on event-store for the creation of a api client", async () => {
+    const { client } = await authorizationService.createApiClient(
+      clientSeed,
+      unsafeBrandId(userId),
```

Comment:
```suggestion
      generateId<TenantId>(),
```
This is the `tenantId` which is different fromt he userId in the seed